src/managers/summary-manager.instrumentation.md

```
@@ -0,0 +1,86 @@
+- `span.commit_story.summary.save_weekly`
+- `span.commit_story.summary.read_month_weeklies`
+- `span.commit_story.summary.save_monthly`
+
+## Function-Level Results
+
+| Function | Status | Spans |
+|----------|--------|-------|
+| readDayEntries | instrumented | 1 |
+| formatDailySummary | instrumented | 0 |
+| saveDailySummary | instrumented | 1 |
+| generateAndSaveDailySummary | skipped — Validation failed: NDS-003, NDS-003, NDS-003, NDS-003 — NDS-003: original line 47 missing/modified: return { saved: false, reason: `Summary already exists for ${dateStr}` };
```

markdownlint-cli2 (0.22.1)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)